raster_api/runtime/setup.py

```diff
@@ -8,7 +8,9 @@
 inst_reqs = [
     "boto3",
     "titiler.pgstac==0.8.0",
-    "titiler.extensions[cogeo]>=0.15.0,<0.16",
+    "titiler.core>=0.15.1,<0.16",
```

Comment:
`titiler.core.dependencies.create_colormap_dependency` was added in `0.15.1`

Reply:
@vincentsarago I noticed the bidx/asset_bidx change in the titiler changelog. Will this [`asset_bidx` parameter usage](https://github.com/NASA-IMPACT/veda-config/blob/main/datasets/frp-max-thomasfire.data.mdx#L25) still be supported in v0.15.4+?
```
    sourceParams:
      asset_bidx: cog_default|1
```

Reply:
`asset_bidx` is the good way to go 👍 

it enable more granular band selection. if you endup doing `asset_bidx: [asset1|1, asset2|1]` , using `bidx=1` will just be easier to write.

Reply:
Never mind, I just confirmed the current usage is not impacted!